src/DIRAC/Interfaces/Utilities/DCommands.py

```diff
@@ -0,0 +1,658 @@
+#!/usr/bin/env python
+import os.path
+import re
+import uuid
+import stat
+import json
+import random
+from configparser import ConfigParser, NoSectionError, NoOptionError
+
+import DIRAC
+from DIRAC import S_OK, S_ERROR, gConfig, gLogger
+import DIRAC.Core.Security.ProxyInfo as ProxyInfo
+import DIRAC.FrameworkSystem.Client.ProxyGeneration as ProxyGeneration
+from DIRAC.ConfigurationSystem.Client.Helpers import Registry
+from DIRAC.Core.Security import Locations, VOMS
+from DIRAC.Resources.Catalog.FileCatalog import FileCatalog
+from DIRAC.Core.Utilities.PrettyPrint import printTable
+
+
+def error(msg):
+    gLogger.error(msg)
+
+
+def critical(msg, exitCode=-1):
+    gLogger.fatal(msg)
+    DIRAC.exit(exitCode)
```

Comment:
Why suddenly we need a new log level?

Reply:
If I rename the function "exit_with_error" (because that's what it is), would that be OK ? It's used in 6 different commands, so it's a bit of an overhead to change/remove it, for (at least from my point of view) little gain.

Reply:
IMHO it is just making confusion, so please remove it. And remove also `error` at line 20.

Reply:
Done